index.html

```diff
@@ -11,22 +11,30 @@
         integrity="sha512-L+sMmtHht2t5phORf0xXFdTC0rSlML1XcraLTrABli/0MMMylsJi3XA23ReVQkZ7jLkOEIMicWGItyK4CAt2Xw=="
         crossorigin="anonymous" referrerpolicy="no-referrer" />
     <script src="https://kit.fontawesome.com/eb94992e2c.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
+
     <title>Osugo</title>
 </head>
 
 <body>
     <div class="start">
         <header>
-            <nav class="menu">
-                <div class="toolbar">
-                    <a href="#" id="logo">Joseck</a>
-                    <i class="fa-solid fa-bars" id="menu-icon"></i>
-
+            <nav class="navbar">
+                <div class="tittle">JOSECK</div>
+                <div class="ham">
+                    <span class="bar1"></span>
+                    <span class="bar2"></span>
+                    <span class="bar3"></span>
                 </div>
-
+                <ul class="nav-sub">
+                    <li class="list-item"><a href="#portfolio" class="nav-link">Portfolio</a></li>
+                    <li class="list-item"><a href="#about" class="nav-link">About</a></li>
+                    <li class="list-item"><a href="#contact" class="nav-link">Contact</a></li>
+                </ul>
```

Comment:
- [ Optional ] Good job implementing the changes from the previous reviewer, however, I've noticed that your implementation of the mobile menu introduces a horizontal scroll on the page which is not good practice. I would suggest that you use the `display: none` CSS rule to hide the mobile menu and use `JavaScript` to manipulate the `display` property of the menu when the hamburger button is clicked. I experienced the error below when using your App.


https://user-images.githubusercontent.com/16392046/223570892-8d55bee6-e525-481c-9c50-ac5c822979af.mp4

Reply:
Thank you so much for this feedback. I had not noticed that.thanks